core/jazz_cloud-logs-streamer/test/test.js

```diff
@@ -1,5 +1,5 @@
-// =========================================================================
-// Copyright © 2017 T-Mobile USA, Inc.
+
+// =========================================================================// Copyright © 2017 T-Mobile USA, Inc.
```

Comment:
PR says "core code change" but I see all the test's here. Can you make sure that you are checking the PR state once submitted?